src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react';
+import { PageNavLink } from '../PageNavLink';
+
+export const NavBar: FC = () => (
+  <nav
+    className="navbar is-light is-fixed-top is-mobile has-shadow"
+    data-cy="Nav"
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <PageNavLink
+          path="/"
+          text="Home"
+          className="navbar-item"
+        />
+
+        <PageNavLink
+          path="/tabs"
+          text="Tabs"
+          className="navbar-item"
+        />
```

Comment:
You can keep data for nav links in the constant and use map() method to not repeat the same logic. Here is my suggestion: 
```
const navLinks = [
  { path: '/', title: 'Home' },
  { path: '/tabs', title: 'Tabs' },
];

Reply:
Good one, thanks)